### ISSUE TITLE
[Bug]: Incorrect PROXY_URL in script.js for multimodal-live-api

### ISSUE THREAD

**Issue by heiko-hotz** [GoogleCloudPlatform/generative-ai#1567]
### File Name

gemini/multimodal-live-api/websocket-demo-app/frontend/script.js

### What happened?

The proxy URL is incorrect. For local setup it probably should be 

`const PROXY_URL = "ws://localhost:8080";`

The Cloud Run option describes how to do it there, but if a user only wants to do local setup they won't read that and expect it to work.

I'd recommend either setting a default proxy URL that works for local setup (preferred) or point the user in the README in the section for local setup to the code where they need to make a change.

### Relevant log output

```shell
window.addEventListener("load", (event) => {
    console.log("Hello Gemini Realtime Demo!");

    setAvalibleCamerasOptions();
    setAvalibleMicrophoneOptions();
});


const PROXY_URL = "wss://[THE_URL_YOU_COPIED_WITHOUT_HTTP]";
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by heiko-hotz** [comment]
please note that #1569 only fixes #1568 . this is a separate issue

**Comment by sartq333** [comment]
yes, that's still 8000 instead of 8080 (https://github.com/GoogleCloudPlatform/generative-ai/tree/main/gemini/multimodal-live-api/websocket-demo-app). after some debugging i figured out that it should be 8080 instead of 8000.

![Image](https://github.com/user-attachments/assets/3b457b49-64ce-4692-bc41-c7ae65196833)
